### PR TITLE
Docs Reorganization

### PR DESCRIPTION
**Pull request by avaidyanatha**
This completes most (if not all) of the docs reorganization that we have planned. There are a few things that we should review before we merge this:

- This required a lot of redirects (as most reorgs do), so we want to make sure that all of them are correct.
- Check the netlify preview (or check this branch out locally) to make sure that we like the new docs organization.
- I've pulled in some new icons from our design system (@dogmar is there a formatter I need to run?)
- I've refactored some old directories to be in line with our new naming, do a sanity check to make sure those are correct.

### CODE REVIEW

**Comment by swoodward90** [comment]
For things that are getting renamed (e.g., Setting up GitOps will eventually live under Managing Your Git Repository), are you planning on people making those changes when they create the new pages?

One other - was Setting Up Third Party DNS going to live under Networking?

**Comment by dogmar** [comment]
> * I've pulled in some new icons from our design system (@dogmar is there a formatter I need to run?)

Nothing extra you need to do. Looks good!

File: README.md [hunk 1]
```diff
@@ -35,6 +35,45 @@ We have created a Cloud Shell with all of the tools and dependencies needed to r
...
+* Plural CLI and Git SCM to maintain the state of a user's applications
+* Plural console for management of all plural applications on your infrastructure
+
+At a high level, the interactions between all three components look something like this:
+
+![](</assets/reference/architecture.png>)
+
+### Plural API
+
+The primary responsibility of the Plural API is to store the packages needed for application installation - terraform, helm - and ingesting high-level dependency information about them. This allows us to properly sequence installations. It also serves as a publish-subscribe layer to communicate updates to clusters that have installed those applications, and can leverage the dependency information ingested to delay updates until a cluster has caught up with all the necessary dependencies.
```
Comment (dherault) [inline README.md:52]: This is heavy to read. IMO could be simplified.

**Review by dherault — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (avaidyanatha) [reply README.md:52]: agreed, this is a bit out of scope, so I'll do this in another PR.

**Pull request merged**